### DISCUSSION TITLE
Filter Shop Packs on Dashboard by Associated Cluster

### DISCUSSION THREAD

**Discussion by elusive-ttv** [gameserverapp/Platform#525] [Suggestions]
It would be amazing to be able to filter shop packs on the dashboard side, by the cluster they are associated with. Having many clusters makes the shop pack list a bit overwhelming, and I feel that by being able to filter by cluster would help to stay organized and up to date with what packs are available without extensive searching through a massive list.

**Comment by mvaessen** [comment]
A development item was created for this request: #612

**Comment by mvaessen** [comment]
This is now available. Thanks for the suggestion!
